### DISCUSSION TITLE
登录失败，建议升级最新版本后重试

### DISCUSSION THREAD

**Discussion by metorayino** [nonebot/discussions#400] [问答]
07-20 01:14:25 [INFO] nonebot_plugin_gocqhttp | [2324565547] 当前版本:v1.1.0
07-20 01:14:25 [INFO] nonebot_plugin_gocqhttp | [2324565547] 将使用 device.json 内的设备信息运行Bot.
07-20 01:14:25 [WARNING] nonebot_plugin_gocqhttp | [2324565547] 警告: 未配置签名服务器, 这可能会导致登录 45 错误码或发送消息被风控
07-20 01:14:25 [INFO] nonebot_plugin_gocqhttp | [2324565547] 开始尝试登录并同步消息...
07-20 01:14:25 [INFO] nonebot_plugin_gocqhttp | [2324565547] 使用协议: Android Pad 8.9.63.11390
07-20 01:14:25 [INFO] nonebot_plugin_gocqhttp | [2324565547] Protocol -> connect to server: 111.30.187.245:8080
07-20 01:14:25 [WARNING] nonebot_plugin_gocqhttp | [2324565547] 登录需要滑条验证码, 请验证后重试.
07-20 01:14:25 [WARNING] nonebot_plugin_gocqhttp | [2324565547] 请选择提交滑块ticket方式:
07-20 01:14:25 [WARNING] nonebot_plugin_gocqhttp | [2324565547] 1. 自动提交
07-20 01:14:25 [WARNING] nonebot_plugin_gocqhttp | [2324565547] 2. 手动抓取提交
07-20 01:14:25 [WARNING] nonebot_plugin_gocqhttp | [2324565547] 请输入(1 - 2)：
07-20 01:14:33 [WARNING] nonebot_plugin_gocqhttp | [2324565547] 请前往该地址验证 -> https://ssl.captcha.qq.com/template/wireless_mqq_captcha.html?style=simple&aid=16&uin=2324565547&sid=7492105768460098033&cap_cd=AGrWg_9brwIC2Dk5ICEoMq8adTAB9P7-izUmOxhY4tgDfe4UJ6ipug**&clientype=1&apptype=2
07-20 01:14:33 [WARNING] nonebot_plugin_gocqhttp | [2324565547] 请输入ticket： (Enter 提交)
07-20 01:15:52 [WARNING] nonebot_plugin_gocqhttp | [2324565547] 登录失败: 登录失败，建议升级最新版本后重试，或通过问题反馈与我们联系。 Code: 45
07-20 01:15:52 [WARNING] nonebot_plugin_gocqhttp | [2324565547] 你的账号被限制登录, 请配置 SignServer 后重试
07-20 01:15:52 [INFO] nonebot_plugin_gocqhttp | [2324565547] 按 Enter 继续....
07-20 01:20:27 [INFO] websockets | connection closed
这个是什么问题啊

**Comment by yanyongyu** [comment]
gocq问题请查看gocq issue，与nonebot无关
